src/pages/index.js

```diff
@@ -262,27 +265,29 @@ const Index = ({ data: initialData }) => {
   useEffect(() => {
     setAdUnits(getAdUnits())
   }, [])
-
   // Only render ads if we are on the client side.
   const [shouldRenderAds, setShouldRenderAds] = useState(false)
   useEffect(() => {
     if (isClientSide()) {
       setShouldRenderAds(true)
     }
   }, [])
-
+  const { app, user } = data || {}
+  const [userGobalId] = useState(get(user, 'id', ''))
```

Comment:
* Fix typo "gobal"
* Let's not use empty strings in place of `undefined`, so just exclude the 3rd param of `get` (or, in this case, just use `user.id`)